modules/nf-core/taxpasta/merge/meta.yml

```diff
@@ -0,0 +1,47 @@
+name: "taxpasta_merge"
+description: Standardise and merge two or more taxonomic profiles into a single table
+keywords:
+  - standardise
+  - merge
```

Comment:
Please compare against: https://nf-co.re/modules/taxpasta_standardise

I made a few suggestions during review of `standardise` and should be copied over